test-data/src/main/resources/org/openmrs/module/fhir2/providers/Location_json_patch.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "op": "replace",
+    "path": "/name",
+    "value": "Patched Location"
+  }
```

Comment:
Is it possible to make this and the merge patch mirror one another so that we have some better examples, e.g.,

```
{
  "op": "add",
  "path": "/address"
}
```

Etc.